docs/rules/capitalized-comments.md

```diff
@@ -0,0 +1,240 @@
+
+Here are the supported object options:
+
+* `ignorePattern`: A string representing a regular expression pattern of words that should be ignored by this rule. If the first word of a comment matches the pattern, this rule will not report that comment.
+    * Note that the following words are always ignored by this rule: `["jscs", "jshint", "eslint", "istanbul", "global", "globals", "exported"]`.
+* `ignoreInlineComments`: If this is `true`, the rule will not report on comments in the middle of code. By default, this is `false`.
+* `ignoreConsecutiveComments`: If this is `true`, the rule will not report on a comment which violates the rule, as long as the comment immediately follows a comment which is also not reported. By default, this is `false`.
+
+Here is an example configuration:
+
+```json
+{
+    "capitalized-comments": [
+        "error",
+        "always",
+        {
+            "ignorePattern": "pragma|ignored",
+            "ignoreInlineComments": true
+        }
+    ]
+}
+```
+
+#### `"always"`
+
+Using the `"always"` option means that this rule will report any comments which start with a lowercase letter. This is the default configuration for this rule.
+
+Note that configuration comments and comments which start with URLs are never reported.
+
+Examples of **incorrect** code for this rule:
+
+```js
+/* eslint capitalized-comments: ["error", "always"] */
+
+// lowercase comment
+
+```
+
+Examples of **correct** code for this rule:
+
+```js
+/* eslint capitalized-comments: ["error", "always"] */
+
+// Capitalized comment
+
+// 1. Non-letter at beginning of comment
+
+// 丈 Non-Latin character at beginning of comment
+
+/* eslint semi:off */
+/* eslint-env node */
+/* eslint-disable */
+/* eslint-enable */
+/* istanbul ignore next */
+/* jscs:enable */
+/* jshint asi:true */
+/* global foo */
+/* globals foo */
+/* exported myVar */
+// eslint-disable-line
+// eslint-disable-next-line
+// https://github.com
+
+```
+
+#### `"never"`
+
+Using the `"never"` option means that this rule will report any comments which start with an uppercase letter.
+
+Examples of **incorrect** code with the `"never"` option:
+
+```js
+/* eslint capitalized-comments: ["error", "never"] */
+
+// Capitalized comment
+
+```
+
+Examples of **correct** code with the `"never"` option:
+
+```js
+/* eslint capitalized-comments: ["error", "never"] */
+
+// lowercase comment
+
+// 1. Non-letter at beginning of comment
+
+// 丈 Non-Latin character at beginning of comment
+
+```
+
+#### `ignorePattern`
+
+The `ignorePattern` object takes a string value, which is used as a regular expression applied to the first word of a comment.
+
+Examples of **correct** code with the `"ignorePattern"` option set to `"pragma":
+
+```js
+/* eslint capitalized-comments: ["error", "always", { "ignorePattern": "pragma" }]
```

Comment:
Oh also, I missed it before submitting this review but there's also a missing `` ` `` after "pragma" right before this code block.